src/pages/News/container/newsContainer.jsx

```diff
@@ -0,0 +1,157 @@
+    }
+  }, [itemActive]);
+
+
+  const items = [
+    {
+      image: "https://newstreettech.com/wp-content/uploads/2024/03/image.jpg",
+      title: 'Slider 01',
+      date: '02-04-2023',
+      description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, neque? Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsum, ex.',
+    },
+    {
+      image: "https://newstreettech.com/wp-content/uploads/2023/12/image-1.png",
+      title: 'Slider 02',
+      date: '02-04-2023',
+      description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, neque? Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsum, ex.',
+    },
+    {
+      image: "https://newstreettech.com/wp-content/uploads/2023/11/Dhanlaxmi-Bank-Inaugurates-Shecommerz-State-Headquarters-powered-by-New-Street-Tech-2000-1.jpg",
+      title: 'Slider 01',
+      date: '02-04-2023',
+      description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, neque? Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsum, ex.',
+    },
+    {
+      image: "https://newstreettech.com/wp-content/uploads/2023/11/Emerging-Businesses-5th-Edition.jpg",
+      title: 'Slider 03',
+      date: '02-04-2023',
+      description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, neque? Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsum, ex.',
+    },
+    {
+      image: "https://newstreettech.com/wp-content/uploads/2023/07/image3.jpg",
+      title: 'Slider 04',
+      date: '02-04-2023',
+      description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, neque? Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsum, ex.',
+    },
+  ];
+  
+  return (
+    <>
+        <header>
+        <Navbar/>
+        </header>
+        <Grid container justifyContent="center">
+        
+        <Grid item xs={12} sm={12} md={12} lg={12} xl={12}>
+          <div className='slider'>
+          
+            <div className='controls'>
+              <div className='control-left'>
+                <Button id="prev" onClick={prevClick}><ArrowBackIosIcon /></Button>
+              </div>
+              <div className='control-right'>
+                <Button id="next" onClick={nextClick}><ArrowForwardIosIcon /></Button>
+              </div>
+            </div>
+ 
+            <div className='thumbnail' ref={thumbnailRef}>
+              {items.map((item, index) => (
+                <div
+                  key={index}
+                  className={`item item-${index} ${index === itemActive ? 'active' : ''}`}
+                  onClick={() => thumbnailClick(index)}
+                >
+                  <img src={item.image} alt={`Slider ${index + 1}`} />
+                  <div className='content'>
+                    <Typography style = {{fontFamily : "Arial", fontSize : "1.5rem"}}>{item.date}</Typography>
```

Comment:
can you make the typography better so that it gets more readable and maybe position at the top

Reply:
ok